Readme.md

```diff
@@ -77,6 +77,8 @@ properties for consistency.
 
 ## AST
 
+Interatively explore the AST with [http://iamdustan.com/reworkcss_ast_explorer/](http://iamdustan.com/reworkcss_ast_explorer/)
```

Comment:
Added. I always go back and forth on periods after URLs for one-off statements. Granted, in proper english that’s not a debate :)